charts/coredns/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: coredns
-version: 1.35.1
+version: 1.35.2
```

```suggestion
version: 1.36.0
```

Also, please update the changes under the annotations section